modules/masterbar/masterbar.php

```diff
@@ -563,6 +563,20 @@ public function add_my_sites_submenu( $wp_admin_bar ) {
 			) );
 		}
 
+		// Site Preview
+		if ( is_admin() ) {
+			$wp_admin_bar->add_menu( array(
+				'parent' => 'blog',
+				'id'     => 'site-preview',
+				'title'  => __( 'Site Preview' ),
```

Comment:
Stephane puts it in eloquent words here
https://core.trac.wordpress.org/ticket/30724#comment:12

> One problem with not escaping translations is that some plugins that filter translations will allow end users to push translations in the back end. 
Depending on where these go they can break the code because characters are not escaped properly and the end users doing the translations won't understand what broke.
This doesn't even take under account that you may not be able to trust the end users doing the translations.
I feel that escaping protects against user error and potentially malicious users especially with translation plugins with little cost / negative repercussions.
Thanks!

But anyway, don't want to block you for this so let's go ahead.